.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,15 +1,22 @@
 ---
 name: Bug Report
 about: Report a bug/crash
-title: ''
+title: '[Bug] A brief description about problem you found'
 labels: ''
 assignees: ''
 
 ---
 
+<!--
```

Comment:
```suggestion

```

1. This makes all markdown in this text useless, since it shows no preview.
2. The likely reason someone wouldn't delete the template is because the used it instead. This just recently happened. In such a case, there would be no description, as it would have been discarded upon posting. I'd rather just edit their comment than them have to re-create it.